vehicle/ford/identity.go

```diff
@@ -75,7 +83,7 @@ func (v *Identity) login() (*oauth.Token, error) {
 	cv := oauth2.GenerateVerifier()
 
 	state := lo.RandomString(16, lo.AlphanumericCharset)
-	uri := OAuth2Config.AuthCodeURL(state, oauth2.S256ChallengeOption(cv),
+	uri := NewOauth2Config(v.domain).AuthCodeURL(state, oauth2.S256ChallengeOption(cv),
```

Comment:
```suggestion
    OAuthConfig := OAuth2Config(v.domain)
	uri := OAuthConfig.AuthCodeURL(state, oauth2.S256ChallengeOption(cv),
```

save the OAuthConfig in a local var, to prevent reassigning in line 184